qa/L1_pytorch_mcore_integration/test.sh

```diff
@@ -8,13 +8,27 @@ set -e
 : ${TE_PATH:=/opt/transformerengine}
 : ${MCORE_PATH:=${TE_PATH}/qa/L1_pytorch_mcore_integration/Megatron-LM}
 
+# Check whether FP8 is supported
+DEVICE_ARCH=$(nvidia-smi --query-gpu=compute_cap --format=csv,noheader | head -n 1 | sed 's/[^0-9]//g')
+if [[ ${DEVICE_ARCH} -ge 89 ]]; then
+    WITH_FP8=1
+fi
+
 # Download Megatron-LM if needed
 if [ ! -d "${MCORE_PATH}" ]; then
     pushd $(dirname ${MCORE_PATH})
     git clone -b core_r0.9.0 https://github.com/NVIDIA/Megatron-LM.git Megatron-LM
```

Comment:
I guess then we have to manually update this branch. Is this okay?

Reply:
I think there's an argument for just downloading the Megatron-LM `main` branch, but that is orthogonal to this PR.